docs/pages/billing/manage.mdx

```diff
@@ -0,0 +1,262 @@
+  style={{ maxWidth: 720 }}
+/>
+
+- Under **Choose a plan**, select **On-demand plan**. Then, click **Continue to Review** at the bottom of the page.
+
+<ImageSpotlight
+  alt="On Cart, select On-demand plan under Choose a plan and Continue to Review at the bottom of the page in Expo dashboard."
+  src="/static/images/billing/management/billing-06.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- Next, you need to review and confirm your upgrade plan. Verify that the account you're downgrading from is correct (see the _warning_ on this page). After reviewing the details, enable the checkbox and click **Continue to Checkout**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-07.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- A **Checkout** popup opens up where you can confirm all the details. Click **Schedule Downgrade** to schedule a downgrade of your plan.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-08.png"
+  style={{ maxWidth: 280 }}
+/>
+
+- After confirming your account for a plan downgrade to On-demand, the same information is also reflected under **Billings** > **Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-09.png"
+  style={{ maxWidth: 640 }}
+/>
+
+</Tab>
+
+<Tab label="From Production and Enterprise to Free plan">
+
+Downgrading from a Production or an Enterprise plan to the Free plan takes effect after your current billing cycle ends.
+
+To downgrade, go to **Billing** and follow the steps below:
+
+- Under **Plan**, click **Change Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the Change Plan button to downgrade a plan."
+  src="/static/images/billing/management/billing-05.png"
+  style={{ maxWidth: 720 }}
+/>
+
+- Under **Choose a plan**, select **Free plan**. Then, click **Continue to Review** at the bottom of the page.
+
+<ImageSpotlight
+  alt="On Cart, select Free plan under Choose a plan and Continue to Review at the bottom of the page in the Expo dashboard."
+  src="/static/images/billing/management/billing-10.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- Next, you need to review and confirm your upgrade plan. Verify that the account you're downgrading is correct (see the _warning_ on this page). After reviewing the details, enable the checkbox and click **Cancel Paid Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-11.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- You will be redirected to Stripe's cancel plan screen, where you can confirm your plan's confirmation.
```

Comment:
Let's add an explainer here that switching to the Free plan is equivalent to canceling at the end of the billing period. It's not obvious from the previous steps that this is what happens.